coverage.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+echo "Executing coverage analysis script"
+# Programs below must be executed for gcov to generate .gcda files
+./src/tlog/tlog-rec-session -c whoami
+./src/tlog/tlog-rec
+./src/tlog/tlog-play
```

Comment:
Coverage data by itself is worthless. Coverage data is a (rough) metric of how much of your program's functionality is being tested. Just starting the programs and not checking what they do, not even checking what their exit status is, would produce some coverage data (i.e. we executed some code) but it wouldn't mean that we tested anything, because we haven't actually checked the outcome.

So, please remove these three lines, write some actual tests, e.g. with Scott's help, and make them run in Travis CI.